Sources/StreamChat/Workers/MessageUpdater.swift

```diff
@@ -563,11 +563,16 @@ class MessageUpdater: Worker {
         })
     }
     
-    func search(query: MessageSearchQuery, completion: ((Error?) -> Void)? = nil) {
+    func search(query: MessageSearchQuery, policy: UpdatePolicy = .merge, completion: ((Error?) -> Void)? = nil) {
```

Comment:
Any reason for the default argument? Since this is an internal API, the default argument might hide an important decision in a Higher Level API.

Reply:
To mirror `UserListUpdater`, no strong reason here